pkgs/tools/misc/env-lscolors/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation {
+  pname = "env-lscolors-unstable";
+  version = "2022-05-05"; # upstream don't do releases
+
+  src = fetchFromGitHub {
+    owner = "trapd00r";
+    repo = "LS_COLORS";
+    rev = "7271a7a8135c1e8a82584bfc9a8ebc227c5c6b2b";
+    sha256 = "0gs2qmdxvvgs5ck2j8b6i8dqc5q91m8xrvc2ajvlhcr7in0n9iw5";
+  };
+
+  # This is unique situation, so it is not really clear how to format and
+  # where to put result.
+  #
+  # Dircolors can output scripts source-able by different shells, but
+  # including them all is redundant and wastes output size; blessing one
+  # particular shell will make users of rest unhappy.
+  #
+  # Here is compromise: value of the variable is put into file, and user is
+  # expected to do something like LS_COLORS=$(cat ...). This is also
```

Comment:
I think we want to install https://github.com/trapd00r/LS_COLORS/blob/master/lscolors.sh to locations where shell automatically sources them like profile.d or the equivalent for zsh.

Reply:
There are a lot of different shells around. And you don't necessary want to include this file unconditionally.

Actually, if you are on 8-color terminal, you definitely don't want to get it included automatically.

Reply:
> There are a lot of different shells around.

we can start with bash and if someone needs another symlink it can be added later one.

> Actually, if you are on 8-color terminal, you definitely don't want to get it included automatically.

I don't think that is an actual real world issue you would be facing when you actively installed this package. I can't even name you a terminal which only support 8 colors. Even the default tty without any GUI has 256 color support.

Reply:
No. 256 colors theme looks horrible on linux (real Ctrl-Alt-F1 terminal). Plus, it is just wrong. If you  `nix-env -iA ssh-agent` it does not mean you want ssh-agent now start automatically on every graphical and console login.

What I can do is define following function:
```
lscolors_init() {
    export LS_COLORS=`cat @out@/share/lscolors/LS_COLORS`
}
```
and have user to call it when and if he deems it necessary.

Reply:
Anyway, added definition of `lscolors_init` that should be good for all Bourne-compatible shells.

Reply:
@SuperSandro2000 Are we missing anything?

Reply:
> What so horrible about `cat`?

that starting a subshell for your profile is really slow compared to just sourcing a file and it is not necessary at all here.